databricks/sdk/_widgets/__init__.py

```diff
@@ -0,0 +1,15 @@
+import os
+
+try:
+    from IPython.core.getipython import get_ipython
```

Comment:
what about working in DBR?

Reply:
This works in dbr

Reply:
@xiaochen-db we do not use the this version (the oss version) of dbutils within dbr at all (https://github.com/databricks/databricks-sdk-py/pull/96). So we shouldn't need to override widgets specifically. Same for ipywidget versioning. The oss implementation is activated only when outside dbr (when a default global implementation of dbutils is not available).